x-pack/plugins/endpoint/server/routes/alerts/list/lib/index.ts

```diff
@@ -108,6 +108,8 @@ export async function mapToAlertResultList(
 
   function mapHit(entry: AlertHits[0]): AlertData {
     return {
+      // base64 encode the alert index in the id field here
+      // https://github.com/elastic/endpoint-app-team/issues/311
```

Comment:
would you please remove this comment and instead link to this line of code in the issue.